stock_request/models/stock_request_abstract.py

```diff
@@ -94,14 +94,14 @@ def _compute_product_qty(self):
         "res.company", "Company", required=True, default=lambda self: self.env.company
     )
     route_id = fields.Many2one(
-        "stock.location.route",
+        "stock.route",
         string="Route",
         domain="[('id', 'in', route_ids)]",
         ondelete="restrict",
     )
 
     route_ids = fields.Many2many(
-        "stock.location.route",
+        "stock.route",
```

Comment:
This is changing the name of the m2m column, so you should take care in a migration script or explicitly keep the old name introducing the argument in the field definition. Anyway, you should see the impact of each of the inherited models.

Reply:
It is a non-store compute, so I understand it is not necessary.